packages/private/edtr-io/src/plugins/exercise.tsx

```diff
@@ -120,6 +120,13 @@ function ExerciseEditor({ editable, state }: ExerciseProps) {
   const store = useScopedStore()
   const { content, interactive } = state
   const [showOptions, setShowOptions] = React.useState(false)
+  const [focusInteractive, setFocusInteractive] = React.useState(false)
```

Comment:
- Why do we need a state?
- Why is the state never reseted after having been set to `true` once?